### PR TITLE
Update the repository version in the Space ROS Dockerfiles

### PR DESCRIPTION
**Pull request by mjeronimo**
Adapt to changes in various repos, bringing things up to date.
The MoveGroup demo has also moved from the MoveIt2 repository
to the MoveIt2 tutorials repository, so also update the READMEs
accordingly as well.

Signed-off-by: Michael Jeronimo <michael.jeronimo@openrobotics.org>

### CODE REVIEW

File: moveit2/README.md
Comment (nuclearsandwich) [inline moveit2/README.md:108]: Was dropping all the content after this initial sentence intentional?

**Review by nuclearsandwich — COMMENTED** [review]
Overall looking good. One fairly significant question raised regarding documentation which I think needs addressing before approval.

File: moveit2/README.md
Comment (mjeronimo) [reply moveit2/README.md:108]: @nuclearsandwich  Yes, the maintainers of the MoveIt2 repository have moved out some of the sample apps into the moveit2_tutorials repo. In particular, the move_group demo that I was using before has been eliminated. So, this section is now obsolete. In the moveit2_tutorials README, I've added a different test to run and have pointed to the online documentation for the details of running the test.

Comment (nuclearsandwich) [reply moveit2/README.md:108]: Thanks for elaborating!

**Review by nuclearsandwich — APPROVED** [review]

**Pull request merged**
